test/e2e/README.md

```diff
@@ -0,0 +1,83 @@
+# Thunderbird Appointment E2E Tests
+
+Guide for running the Thunderbird Appointment E2E tests.
+
+## Prerequisite
+
+You must have a pre-existing Appointment user test account (using FxA credentials) on the platform where you are running the tests. ie. For the production sanity test you must have an Appointment test account on production (using production FxA credentials) already set up.
+
+## Installation
+
+First install the E2E suite (includes Playwright):
+
+```bash
+cd test/e2e
+npm install
+```
+
+Next install the Playwright browsers (Playwright uses it's own bundled browers) still in `test/e2e`:
+
+```bash
+npx playwright install
+```
+
+## Running Locally
+
+The E2E tests require credentials for an existing Appointment (FxA) account and reads these from your local env vars. First copy over the provided `.example.env` to a local `.env`:
+
+```bash
+cd test/e2e
+cp .env.example .env
+```
+
+Then edit your local `.env` file and provide the credentials for your Appointment test account:
+
+`APPT_PROD_LOGIN_EMAIL=<existing-test-FxA-user-email>`</br>
+`APPT_PROD_LOGIN_PWORD=<exisiting-test-FxA-user-password>`
+
+To run the production sanity test headless (still in `test/e2e`):
+
+```bash
+npm run prod-sanity-test
+```
+
+To run the production sanity test with a UI so you can watch the tests run (still in `test/e2e`):
+
+```bash
+npm run prod-sanity-test-ui
+```
+
+To run the production sanity test in debug mode (still in `test/e2e`):
+
+```bash
+npm run prod-sanity-test-debug
+```
+
+## Running on BrowserStack
+
+You can run the E2E tests from your local machine but against browsers provided in the BrowserStack Automate cloud.
+
+<b>For security reasons when running the tests on BrowserStack I recommend that you use a dedicated test Appointment FxA account / credentials (NOT your own personal Appointment (FxA) credentials).</b>
+
+Once you have credentials for an existing Appointemnt test account, edit your local `.env` file and add the credentials:
+
+`APPT_PROD_LOGIN_EMAIL=<existing-test-FxA-user-email>`</br>
+`APPT_PROD_LOGIN_PWORD=<exisiting-test-FxA-user-password>`
+
+Also in order to run on BrowserStack you need to provide your BrowserStack credentials. Sign into your BrowserStack account and navigate to your `User Profile` and find your auth username and access key. In your local terminal export the following env vars to set the BrowserStack credentials that the tests will use:
+
+```bash
+export BROWSERSTACK_USERNAME=<your-browserstack-user-name>
+```
+
+```bash
+export BROWSERSTACK_ACCESS_KEY=<your-browserstack-access-key>
```

Comment:
Can these not be moved to the dotenv?

This is mentioned for selenium but I can't imagine it wouldn't work for playwright 😅 
https://www.browserstack.com/docs/automate/selenium/sdk-faqs/nodejs/load-env-var

Reply:
Hmmm I have the .env file for FxA creds but didn't think that was possible to put the BrowserStack creds in there because when using BrowserStack you actually call a BrowserStack command first and provide the command that is used in-turn to run the PlayWright tests. So the dotenv for the E2E PlayWright tests isn't even called yet when BrowserStack is called. I'll give that a try though and see if passing in the .env works and if BrowserStack can read it's own credentials from there when it is called. Thanks